### ISSUE TITLE
Error: Cannot find module 'azure-***s-task-lib/task'

### ISSUE THREAD

**Issue by SimonAlling** [jfrog/artifactory-azure-devops-extension#149]
[Release 1.9.1](https://github.com/jfrog/artifactory-azure-devops-extension/commit/2638c56b06281cb8784467943dfc079d1d12e7e4) seems to have broken our pipeline.

```
Starting: Push Docker image to Artifactory
==============================================================================
Task         : Artifactory Docker
Description  : Pull and push docker images from and to Artifactory, while allowing to collect build-info. The collected build-info can be later published to Artifactory by the "Artifactory Publish Build Info" task.
Version      : 1.9.1
Author       : JFrog
Help         : [More Information](https://www.jfrog.com/confluence/display/RTF/Docker+Registry)
==============================================================================
module.js:478
    throw err;
    ^

Error: Cannot find module 'azure-***s-task-lib/task'
    at Function.Module._resolveFilename (module.js:476:15)
    at Function.Module._load (module.js:424:25)
    at Module.require (module.js:504:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/vsts/work/_tasks/ArtifactoryDocker_07fc5c20-6882-a1c2-cbcd-6531d5a72113/1.9.1/dockerBuild.js:1:74)
    at Module._compile (module.js:577:32)
    at Object.Module._extensions..js (module.js:586:10)
    at Module.load (module.js:494:32)
    at tryModuleLoad (module.js:453:12)
    at Function.Module._load (module.js:445:3)
##[error]Exit code 1 returned from process: file name '/home/vsts/agents/2.165.2/externals/node/bin/node', arguments '"/home/vsts/work/_tasks/ArtifactoryDocker_07fc5c20-6882-a1c2-cbcd-6531d5a72113/1.9.1/dockerBuild.js"'.
Finishing: Push Docker image to Artifactory
```

Pipeline configuration:

```yaml
- task: ArtifactoryDocker@1
  displayName: Push Docker image to Artifactory
  condition: >
    and(
      succeeded(),
      or(
        ne(variables['DONT_PUSH_TO_ARTIFACTORY'], 'true'),
        eq(variables['Build.SourceBranch'],'refs/heads/master')
      )
    )
  inputs:
    command: push
    artifactoryService: Artifactory
    targetRepo: docker
    imageName: $(IMAGE_NAME):$(Build.BuildId)
    collectBuildInfo: true
    buildName: $(Build.DefinitionName)
    buildNumber: $(Build.BuildNumber)
    includeEnvVars: true
```

**Comment by yahavi** [comment]
Thanks for reporting this issue @SimonAlling!
We're on it.

**Comment by eyalbe4** [comment]
All - we believe that the issue is resolved now.
Can you please confirm?

**Comment by lindhe** [comment]
Is there any way to pin the version of the task in DevOps, to avoid updates like these?

**Comment by cubimon** [comment]
> All - we believe that the issue is resolved now.
> Can you please confirm?

It works now

**Comment by lindhe** [comment]
Yeah, works here too.

**Comment by eyalbe4** [comment]
@lindhe,
As far as I know there isn't. As soon as a new version is published, all environments are updated automatically.
The cause of this issue was a bug in the extension's release pipeline, which we identified and fixed. We immediately published patch 1.9.2, which is built correctly and seems to have resolved the issue.
We take this incident very seriously and truly apologies for the inconvenience it caused you.
Thank you all for using the Artifactory Extension. We work very hard to continue improving it.

**Comment by SimonAlling** [comment]
Thank you for the swift response!
